samples/tab-sso/nodejs/config/default.json

```diff
@@ -1,8 +1,8 @@
 {
-    "tab": {
-        "appId": "MICROSOFT_APP_ID",
-        "appPassword": "MICROSOFT_APP_PASSWORD",
-        "applicationIdUri": "Application_Id_Uri"
-    },
-    "port": "3333"
-}
+  "tab": {
+    "appId": "MICROSOFT_APP_ID",
+    "appPassword": "MICROSOFT_APP_PASSWORD",
```

Comment:
What is this property being used for?

Reply:
This property is used for oauth flow to exchange id token with access token.
https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/929293c0542ef090f2dfb90b2f332906c5f750f2/samples/tab-sso/nodejs/src/server/tabs.js#L55

Reply:
Ah OK - the naming threw me off. `clientSecret` is clearer to me.

Could we rename this property in the `default.json` to convey that this is the client secret? It would also be a good idea to include a comment that we do not recommend developers check this value into their repo, but rather store it in some secure location like Key Vault

Reply:
Yes changed the property name and added one note in readme for not pushing the app credentials.